i18n/locale-en.json

```diff
@@ -0,0 +1,265 @@
+{
+  "I18N_GALLERY_PAGE_TITLE": "Oppia - Gallery",
+  "I18N_GALLERY_TITLE": "Hi, I'm Oppia",
+  "I18N_GALLERY_SUBTITLE": "Your personal tutor for ",
```

Comment:
This is a basic i18n mistake. Whatever follows should be a variable in here, and not be glued to this string.